### ISSUE TITLE
Populate `Owner` tag in ListSubscriptionsByTopic Response

### ISSUE THREAD

**Issue by Admiral-Piett** [Admiral-Piett/goaws#257]
In the `ListSubscriptionsByTopic` Response there's an owner field which, in aws is populated with the user's account ID (no dashes).  In our app, there's nothing populated there.

Additional Questions:
* Are there other places we don't populate this field as well?

To Recreate:
![Screen Shot 2023-03-14 at 9 40 22 AM](https://user-images.githubusercontent.com/49883489/225020074-bc4c997d-e8f1-445b-a8e5-711995872d2d.png)

AWS Request:
```
aws sns list-subscriptions-by-topic --topic-arn arn:aws:sns:us-east-1:<account_id>:TOPIC_1 --no-verify-ssl
```

**Comment by Admiral-Piett** [comment]
@chrisdaly3 This might be a good one to check out if you're interested and have the time.

**Comment by chrisdaly3** [comment]
> @chrisdaly3 This might be a good one to check out if you're interested and have the time.

Thanks - I'll assign and take a look this week!

**Comment by Admiral-Piett** [comment]
@chrisdaly3 - I know you wanted to talk this through a WHILE ago.  I'm so sorry.  I got sidetracked on it.  Let's sync up here soon and get this one over the line?

**Comment by chrisdaly3** [comment]
@Admiral-Piett sounds like a plan to me. I've got some time later tomorrow afternoon or we can catch up on weeknight/etc. Just let me know 😸

**Comment by chrisdaly3** [comment]
Resolved in #287 🚀
